### PR TITLE
feat: add conversion of feed flow between units

### LINKED ISSUE #80: Add support for feed in kg/d
To feedFlow input

### PR DESCRIPTION
**Pull request by mariuswinger**
## Why is this pull request needed?
Add support for conversion of feed units.

## What does this pull request change?
User can now convert between units and the conversion will happen in the UI. Unit switch is disabled until a composition feed is selected.

## Issues related to this change:
Closes #80

### CODE REVIEW

File: web/src/types.ts [hunk 1]
```diff
@@ -12,4 +12,5 @@ export type TPackage = {
   name: string
   description: string
   components: TComponentRatios
+  molecularWeight: number
```
Comment (ingeridhellen) [inline web/src/types.ts:15]: Isn't molecular weight a constant of the chemical compound? If so, I don't think it makes sense to store it as part of the package

File: web/src/pages/Main.tsx [hunk 2]
```diff
@@ -37,10 +37,8 @@ export const MainPage = (props: { mercuryApi: MercuryAPI }): JSX.Element => {
   const [componentProperties, setComponentProperties] =
     useState<TComponentProperties>()
   const [isLoading, setIsLoading] = useState<boolean>(true)
-  const [feedFlow, setFeedFlow] = useState<TFeedFlow>({
```
Comment (ingeridhellen) [inline web/src/pages/Main.tsx:40]: Does that mean we can remove the TFeedFlow type?

File: web/src/types.ts (hunk 1 above)
Comment (ingeridhellen) [reply web/src/types.ts:15]: Ah, no, okay, I get it. Could we perhaps call it something else then? combinedMolecularWeight, molecularWeightSum or something like that? I see you call it feedMolecularWeight in FeedFlowInput, but I'm not sure that is better. Best if it's called the same everywhere, though

File: web/src/components/feature/FeedFlowInput.tsx [hunk 3]
```diff
@@ -1,40 +1,72 @@
...
+import { molePerStandardCubicMeter } from '../../constants'
+import { useState } from 'react'
 
 const FlexContainer = styled.div`
   display: flex;
   gap: 16px;
   max-width: 250px;
 `
 
+function convertCubicFeedFlowValue(
```
Comment (ingeridhellen) [inline web/src/components/feature/FeedFlowInput.tsx:12]: I know I send in cubicFeedFlow. The question is what I get out. Perhaps rename to `getMassFeedFlow` or something like that, and similarly for the function below

File: web/src/types.ts (hunk 1 above)
Comment (mariuswinger) [reply web/src/types.ts:15]: I agree. Will rename it.

File: web/src/pages/Main.tsx (hunk 2 above)
Comment (mariuswinger) [reply web/src/pages/Main.tsx:40]: Yes. We assume the feed flow is cubic everywhere and just convert/display it as mass flow if the switch is clicked.

File: web/src/components/feature/FeedFlowInput.tsx (hunk 3 above)
Comment (mariuswinger) [reply web/src/components/feature/FeedFlowInput.tsx:12]: Yeah. Was trying to avoid a very long name, but probably better clarity that way.

**Pull request merged**